### ISSUE TITLE
session_id - 00000000-1111-2222-3333-444444444444

### ISSUE THREAD

**Issue by karol-skibinski** [nigelm/broadworks_ocip#2]
* Broadworks OCI-P Interface version: 16
* Python version: 3.8
* Operating System: Windows 10

### Description

Wrong Session ID in Python output (e.g to_json()). 

### What I Did
When I wrote:
```
...
res_group = api.command("GroupGetRequest14sp7", service_provider_id=my_sp, group_id=my_gr)
j_group = res_group.to_json(indent=2)
print(j_group)
...
```
I have output
```
"session_id": "00000000-1111-2222-3333-444444444444"
```
In XML stream i see:
```
<sessionId xmlns="">af33c996-XXXX-4213-8663-xxxxxxxxxxxx (anonymized)
```
**Both session_id from the same session.**

**Comment by nigelm** [comment]
It took me a while to work out what the bug report was describing.

The session id was previously not being passed into objects from the XML decode.

This has been fixed in commit b5946695e98ccaf7cc0ec7396c7b0482d09bf07b, and a test added to pick this up in 9cd81d94fd1273f930adde435f7819ee5910d6b5

At this point the fixes have not been released as they are in a more extensive set of changes.

**Comment by karol-skibinski** [comment]
I confirm, now (2.0.0) it's ok!
Thanks
